lib/services/local/headlessauthn.go

```diff
@@ -30,6 +30,7 @@ import (
 
 // CreateHeadlessAuthenticationStub creates a headless authentication stub in the backend.
 func (s *IdentityService) CreateHeadlessAuthenticationStub(ctx context.Context, name string) (*types.HeadlessAuthentication, error) {
+	// Stub should be replaced shortly after creation.
```

Comment:
Can you please add more context to how this function works and why we are generating the stub? Reading this without context is hard to understand.